layouts/events/list.html

```diff
@@ -13,7 +13,17 @@ <h1>Upcoming Events</h1>
 
 {{ define "main" }}
     <div class="container mx-auto">
-        <div class="flex justify-between py-10 md:py-24 px-5 md:px-20">
+        <div class="w-10/12 mx-auto mt-10 note note-info" role="alert">
```

Comment:
This doesn't quite seem to line up with the sections below it:

![image](https://user-images.githubusercontent.com/274700/78702814-0880ce80-78be-11ea-9f82-7fef34d8e210.png)

And needs padding to line up properly on mobile (we apply the same padding pretty much everywhere):

![image](https://user-images.githubusercontent.com/274700/78703132-8b098e00-78be-11ea-9d7f-b58303ff0c94.png)